src/TimeNode/index.js

```diff
@@ -11,10 +11,15 @@ const Repl = require('./repl');
 const { checkOptionsForWalletAndPassword } = require('../Wallet/utils');
 
 const timenode = async (options, program) => {
+  if (program.config) {
+    const config = JSON.parse(fs.readFileSync(program.config));
+    program.password = fs.readFileSync(config.password).toString();
+    program.wallet = config.wallet;
+  }
```

Comment:
Can we make it support other parameters besides `wallet` and `password`? To make it truly modular.